docs/further.md

```diff
@@ -136,4 +136,27 @@ rule myrule:
         slurm_extra="--qos=long --mail-type=ALL --mail-user=<your email>"
 ```
 
-Again, rather use a [profile](https://snakemake.readthedocs.io/en/latest/executing/cli.html#profiles) to specify such resources.
\ No newline at end of file
+Again, rather use a [profile](https://snakemake.readthedocs.io/en/latest/executing/cli.html#profiles) to specify such resources.
+
+## Software Recommendations
+
+### Conda, Mamba, Micromamba
+
+While Snakemake mainly relies on Conda for reproducible execution, many clusters impose file number limits in their "HOME" directory. Using [micromamba](https://mamba.readthedocs.io/en/latest/user_guide/micromamba.html) in place of [mamba](https://mamba.readthedocs.io/en/latest/), which is recommended by Snakemake, might be a remedy: Micromamba does not save all the package files, hence users do not have to clean up manually.
```

Comment:
Yes, but we have to first add micromamba support to Snakemake. There is a PR, but it needs to be adapted to Snakemake 8. I will try to finalize that ASAP. The most sustainable approach will be to first create an interface for deployment plugins and then move the PR code into a new plugin.

Reply:
I have been in contact with them. Pixi is not really meant for usage with envs like micromamba/mamba/conda is. It is rather like a generalization of poetry. With one env per "project", and users not really interacting directly with envs. However, pixi builds on rattler, and I think it makes most sense to just use rattler-py (a python api to rattler) in Snakemake directly. I will soon work on the software-deployment plugin interface for snakemake, then move the conda support into a corresponding plugin and offer a rattler option there.

Reply:
> Pixi is not really meant for usage with envs like ...

So I noticed. Still need to wrap my head around this and how the differences can be introduced in courses.

Reply:
The last commit is an attempt to clarify.